pkg/rehearse/jobs_test.go

```diff
@@ -35,12 +35,56 @@ import (
 
 	"github.com/openshift/ci-tools/pkg/api"
 	"github.com/openshift/ci-tools/pkg/config"
+	"github.com/openshift/ci-tools/pkg/jobconfig"
 	"github.com/openshift/ci-tools/pkg/load"
 	"github.com/openshift/ci-tools/pkg/registry"
 )
 
 const testingRegistry = "../../test/multistage-registry/registry"
 
+// configFiles contains the info needed to allow inlineCiOpConfig to successfully inline
+// CONFIG_SPEC and not fail
+var configFiles = config.ByFilename{
```

Comment:
This is a global and if someone does not know that they cannot change it, you could get bugs in test from modification. We could wrap this in a producer method and give each consumer their own version